plugin/storage/integration/integration.go

```diff
@@ -422,4 +476,9 @@ func (s *StorageIntegration) IntegrationTestAll(t *testing.T) {
 	t.Run("GetLargeSpans", s.testGetLargeSpan)
 	t.Run("FindTraces", s.testFindTraces)
 	t.Run("GetDependencies", s.testGetDependencies)
+	if s.SamplingStore == nil {
```

Comment:
this should be done inside the individual tests that depend on sampling store, not in the top runner

Reply:
I also thought of that but then I realized that in all the individual tests we are calling `s.IntegrationTestAll(t)` which will run the integration test for them and we can skip sampling store test at the top runner level. 
If I try to skip it at the individual level I'm not sure how can we skip this two specific sampling store test because there we can check for the samplingStore is nil or not .. but we cannot specify that we want to skip this two test.